x-pack/legacy/plugins/siem/public/pages/detection_engine/detection_engine.tsx

```diff
@@ -35,6 +38,12 @@ import { DetectionEngineEmptyPage } from './detection_engine_empty_page';
 import { DetectionEngineNoIndex } from './detection_engine_no_signal_index';
 import { DetectionEngineUserUnauthenticated } from './detection_engine_user_unauthenticated';
 import * as i18n from './translations';
+import { DETECTION_ENGINE_PAGE_NAME } from '../../components/link_to/redirect_to_detection_engine';
+
+export enum DetectionEngineTab {
+  signal = 'signal',
+  alert = 'alert',
```

Comment:
Consider changing both the constant name `alert`, and the value `'alert'` to the plural form `alerts`, because most other page routes use the plural form. For example, the URL path to the Alerts tab on the hosts page uses the plural form `alerts`:

```
/app/siem#/hosts/alerts
```